.github/workflows/actions.yml

```diff
@@ -127,3 +129,70 @@ jobs:
         grunt test
       env:
         CI: true
+  publish:
+    name: Publish to PyPI registry
+    needs:
+      - build_python
+      - build_javascript
+    runs-on: ubuntu-latest
+
+    env:
+      PY_COLORS: 1
+      TOXENV: packaging
+
+    steps:
+      - name: Switch to using Python 3.6 by default
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```

Comment:
Is there a reason why we're using 3.6 and not a newer version @ssbarnea @gnikonorov ?

Reply:
I'm pretty sure Sorin wanted to support the oldest possible version that the devs/contributors may use.

Reply:
Yes, use of oldest supported is usually better than newest in order to avoid regressions. Based on experience using oldest proved to catch more issues than the newest, especially in this area. 

Any CI failure that prevents a bad release is better than making a user unhappy. Note to myself: add comments on action tasks like this documenting their purpose.

Reply:
Thanks @ssbarnea! Makes sense. 👍